packages/eslint-config-custom/index.js

```diff
@@ -30,6 +30,9 @@ module.exports = {
     'tailwindcss/no-custom-classname': [
       'error',
       {
+        // All of these callees are the Tailwind defaults, except `cn`, which is
+        // our own custom helper.
+        callees: ['classnames', 'clsx', 'cn', 'ctl', 'cva', 'tv'],
```

Comment:
This is one of the main changes to this PR: adding `cn` to the list of `callees` ([docs](https://github.com/francoismassart/eslint-plugin-tailwindcss/blob/master/docs/rules/no-custom-classname.md#callees-default-classnames-clsx-ctl-cva-tv)). This ensures that ESLint will still check our class names when they're wrapped in a `cn()` call.